publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -32,46 +32,145 @@ guide-category: pipelines
+    ```
+    kind: ConfigMap 
+    apiVersion: v1 
+    metadata:
+      name: gitops-map
+      namespace: kabanero
+    data:
+      gitops-repository-url: <can be specified here if common for all the pipelines in the cluster or in the event mediator for specific versions of the pipeline> 
+      gitops-repository-type: <github,gitlab,ghe>
+      gitops-commit-user-name: <user_name_to_commit_using>
+      gitops-commit-user-email: <user_email_to_commit_using>
+    ```
+    
+    A secret called `gitops-token` also has to be created in the Kabanero namspace. This example yaml creates the secret.
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: gitops-token
+    annotations:
+      tekton.dev/git-0: https://github.com
+    namespace: kabanero
+    type: kubernetes.io/basic-auth
+    stringData:
+      username: <gitops_repo_username>
+      password: <gitops_repo_access_token>
+    ```
+       
+  * [deploy-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/events/deploy-task.yaml)
+    If the `webhooks-tekton-local-deploy` property is set to `true` in the mediator, the image is deployed to the namespace configured in the `app-deploy.yaml`.  By default, the application will be deployed in the `kabanero` namespace.
+  
+  * [image-scan-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/events/image-scan-task.yaml)
+    The `image-scan-task` task will initiate a container scan of the image published by the `build-push-task` using OpenSCAP.  The results of the scan are published in the logs of the task.
 
-The default tasks and pipelines are provided in the [pipelines repository](https://github.com/kabanero-io/kabanero-pipelines/tree/master/pipelines/incubator).  Details of some of the primary pipelines and tasks are described in the following information.
+* **A release of the application is created**
 
+This will trigger the `image-retag-pl` pipeline which leverages the [image-retag-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/events/image-retag-task.yaml) to create a new tag of the image to match with the git release.
 
-### The build, push and deploy pipeline
+* **The pull request in the gitops repo is merged** (This feature is Tech Preview in this release)
+When the PR that was created by the promote step of the `build-push-promote-pl` is merged in the gitops repo, it will trigger  the `deploy-kustomize-pl` pipeline, which leverages the [deploy-kustomize-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/events/deploy-kustomize-task.yaml) to trigger a deployment to the environment configured in the gitops repo.
```

Comment:
Not sure about "trigger a deployment" because there could be many deployments. Suggest "apply the updated GitOps state to the environment".